src/box.ts

```diff
@@ -285,7 +285,9 @@ export const stbl = (track: Track) => {
 		stsz(track),
 		stco(track)
 	];
-	if (track.compositionTimeOffsetTable.length > 0) {
+	const needsCTTS = track.compositionTimeOffsetTable.length > 1 ||
+		track.compositionTimeOffsetTable.some((x) => x.sampleCompositionTimeOffset !== 0);
+	if (needsCTTS) {
```

Comment:
Nice check. One consistency thing: The `children` array passed to `box` correctly works with `null` elements, as this is the recommended approach to omit boxes. So you'd do a:
```ts
return box('stbl', null, [
  ...,
  needsCTTS ? ctts(track) : null
])
```

Reply:
Ah, excellent! I wasn't sure if that would work or not.